### ISSUE TITLE
Sidebar item toggle missing text (a11y issue)

### ISSUE THREAD

**Issue by batpigandme** [quarto-dev/quarto-cli#704]
Currently the sidebar item toggle (`.sidebar-item-toggle`) uses the chevron icon, but does not have an accessible text label, e.g.
``` html
<a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" href="">
        <i class="bi bi-chevron-right ms-2"></i>
</a>
```

This can be resolved with an `aria-label`, `aria-labelledby`, etc.
